src/x11/keymap.c

```diff
@@ -832,6 +832,8 @@ get_sym_interprets(struct xkb_keymap *keymap, xcb_connection_t *conn,
         case XCB_XKB_SYM_INTERPRET_MATCH_EXACTLY:
             sym_interpret->match = MATCH_EXACTLY;
             break;
+        default:
+            break;
```

Comment:
Hmm, leaving out the `default` was sort of an "this can't happen", but if we add it, it looks legit, but we can't leave `sym_interpret->match` uninitialized though. It needs to either error, assert or fallback.